pom.xml

```diff
@@ -90,11 +90,11 @@
                module. Use the contents of that for OUR version of startup.properties, but don't
                forget to replace the felix config admin bundle with OUR platform-osgi-configadmin
                bundle if you copy-pasted. -->
-        <karaf.version>4.1.2</karaf.version>
+        <karaf.version>4.1.4</karaf.version>
         <felix.configadmin.version>1.8.14</felix.configadmin.version>
```

Comment:
Do we need to upgrade the configadmin version here? Between Karaf 4.1.2 and Karaf 4.1.4, `felix.configadmin.version` was updated from 1.8.14 to 1.8.16.
https://github.com/apache/karaf/commit/a7f23715b91e0f488ded059817757573e1cd0c72. @Lambeaux

Reply:
...and now I see that my earlier tagging of @Lambeaux has probably already been considered and addressed.